examples/AzureCommunicationChatPushNotificationTest/AzureCommunicationChatPushNotificationTest/AppDelegate.swift

```diff
@@ -0,0 +1,188 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+import UIKit
+import UserNotifications
+import AzureCommunicationChat
+import AzureCommunicationCommon
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    let token = "eyJhbGciOiJSUzI1NiIsImtpZCI6IjEwM19pbnQiLCJ4NXQiOiJlU0t5M1lPd2FzazIxN3ZKWmVWZFhOcDBSNm8iLCJ0eXAiOiJKV1QifQ.eyJza3lwZWlkIjoiYWNzOjE4OGQ0Y2VhLTBhOWItNDg0MC04ZmRjLTdhNWM3MWZlOWJkMF8wMDAwMDAwZi1kZGQxLWM2NGYtZjk5NC0wOTQ4MjIwMDRiYjIiLCJzY3AiOjE3OTIsImNzaSI6IjE2NDYwMTk4NjkiLCJleHAiOjE2NDYxMDYyNjksImFjc1Njb3BlIjoiY2hhdCIsInJlc291cmNlSWQiOiIxODhkNGNlYS0wYTliLTQ4NDAtOGZkYy03YTVjNzFmZTliZDAiLCJpYXQiOjE2NDYwMTk4Njl9.g1GEVs5zKM3uc88G7DpOFbvbJh_1CVIz7uZe6n8G6CxnQ4B2wuqTRI5uXwAdvw7M-qjHf62GUt1dt_DYz7E-HBbj3maZzORlsa6XV6Lut3anSHy8G2DE3HeN1AyXaIahXgHf4tlsC-zpDhNrskr-WYTMMTLBkizXfNjVqPWGhhJxyWtyJSEOSSh-kDWdmWVnyP5jAy1XLC96_NDkIzBsY59BnBlpwuI6EF_rnBZwlKZ-EnFx8c2AgkwWpGp-EHEdPK785i4tl21RSQ8LUhMxwfgQDPhpvbvwu8SJRvzufMwc_OOYZnXmA5kAs8_GkQ8hJqtpU0cp62pF684zEV62bQ"
```

Comment:
While this is an expired token, you **must not** commit credentials to any repo. Please remove this and refactor the code to get credentials in a safe way.